lib/proxi/local/proxi_datasets.py

```
@@ -1182,7 +1156,11 @@ def compute_facets(self, rows):
                     value = value.strip()
                     if facet_name == 'sdrf':
                         if value != '':
-                            value = 'SDRF'
+                            try:
+                                sdrf_source = value.split(':')[0]
+                                value = f"{sdrf_source} SDRF"
+                            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)